folder/README.md

```diff
@@ -8,4 +8,26 @@ We are learning how to make commits.
 
 We are about to make a commit without best practices.
 
-goodbye
\ No newline at end of file
+goodbye
+
+docs
+
+repositories, branches, commits, tags
+
+# created the repo locally
+local repository -> "remote" repository (git add remote <name> <URL>)
+
+# created the repo remotely
```

Comment:
I find this section vafue. Please consider adding a verbal description of what these commands are actually doing.